recipe/meta.yaml

```diff
@@ -18,85 +18,160 @@ source:
     #
     # https://github.com/Homebrew/legacy-homebrew/blob/21ce7a56cb3fc40fe93d731cc654c69268796772/Library/Formula/glib.rb#L33-L39
     - patches/0001-replace-hardcoded-paths.patch
-    #- patches/0001-Manually-link-with-libiconv-whenever-we-use-libintl.patch  # [win]
-    - patches/0002-skip-some-tests.patch                              # [not osx]
-    - patches/0003-add-install_rpath-to-meson-build-files.patch
+    - patches/0002-Manually-link-with-libiconv-whenever-we-use-libintl.patch  # [win]
     - patches/0004-Attempt-to-mask-out-qemu-failing-tests.patch       # [ppc64le or aarch64]
-    - patches/0005-Increase-some-test-timeouts.patch                  # [win]
+    - patches/0006-correct-osx-tests-skips.patch
+    - patches/0007-win-pkgconfig-test-bash-friendly.patch # [win]
+    - patches/0008-skip-tests-needing-system-utils.patch # [linux]
+    - patches/0009-s390x-skip-locale-check.patch # [s390x]
 
 build:
-  number: 2
-  # Python is only ever needed for the build process, so use this next `skip`
-  # statement to avoid redudant builds caused by any cbc.yaml.
-  skip: True  # [py!=38]
-  skip: True  # [win and vc<14]
-  missing_dso_whitelist:          # [osx]
-    - /usr/lib/libresolv.9.dylib  # [osx]
-  run_exports:
-    - {{ pin_subpackage('glib') }}
-  ignore_run_exports:
-    - libffi  # [win]
+  number: 0
+  ignore_run_exports_from:
+    - python *
 
 requirements:
   build:
-    - meson 0.53.2  # [win]
-    - meson         # [not win]
+    - patch # [unix]
+    - {{ posix }}patch  # [win]
+    - meson >=0.60.0
+    - cmake
```

Comment:
why's cmake needed?

Reply:
meson can use both cmake and pkg-config to find libraries.
https://mesonbuild.com/Dependencies.html#dependency-detection-method